tests/misc/tnegativezero.nim

```diff
@@ -0,0 +1,30 @@
+discard """
```

Comment:
@xflywind this is hard to find, IMO we should merge this into tests/system/tdollars.nim (named after dollars.nim)

Reply:
done https://github.com/nim-lang/Nim/pull/16508